articles/ros2_threat_model.md

```diff
@@ -187,6 +253,10 @@ attack).
     <td>Physical Safety</td>
```

Comment:
Do we need "physical" here? Isn't it redundant in this context given the fact that safety typically cares for ensuring these exact aspects?
```suggestion
    <td>Safety</td>
```

Reply:
I would keep it in order to make it more clear.

Reply:
@thomas-moulard what's your view on this?